package.json

```diff
@@ -139,7 +139,7 @@
     "stylelint-declaration-use-variable": "1.7.2",
     "stylelint-processor-styled-components": "1.3.1",
     "superagent": "^3.6.0",
-    "uglifyjs-webpack-plugin": "^2.2.0",
+    "terser-webpack-plugin": "^4.2.3",
```

Comment:
We cannot use latest terser-webpack-plugin here as it is supported only with webpack v5.
This repo uses v4